cluster/examples/kubernetes/ceph/object-multisite.yaml

```diff
@@ -2,44 +2,44 @@
 # Create an object store with settings for a test environment. Only a single OSD is required in this example.
 #  kubectl create -f object-multisite.yaml
 #################################################################################################################
-apiVersion: ceph.rook.io/v1
-kind: CephObjectRealm
-metadata:
-  name: realm-a
-  namespace: rook-ceph # namespace:cluster
----
-apiVersion: ceph.rook.io/v1
-kind: CephObjectZoneGroup
-metadata:
-  name: zonegroup-a
-  namespace: rook-ceph # namespace:cluster
-spec:
-  realm: realm-a
----
-apiVersion: ceph.rook.io/v1
-kind: CephObjectZone
-metadata:
-  name: zone-a
-  namespace: rook-ceph # namespace:cluster
-spec:
-  zoneGroup: zonegroup-a
-  metadataPool:
-    failureDomain: host
-    replicated:
-      size: 1
-      requireSafeReplicaSize: true
-  dataPool:
-    failureDomain: host
-    replicated:
-      size: 1
-      requireSafeReplicaSize: true
-    parameters:
-      compression_mode: none
----
+# apiVersion: ceph.rook.io/v1
```

Comment:
You didn't intend to commit this file, right? looks like some local testing

Reply:
Right....